lib/timers.js

```diff
@@ -389,12 +390,12 @@ const unenroll = exports.unenroll = function(item) {
 // Using existing objects as timers slightly reduces object overhead.
 exports.enroll = function(item, msecs) {
   if (typeof msecs !== 'number') {
-    throw new TypeError('"msecs" argument must be a number');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'msecs',
+                               'number', msecs);
   }
 
   if (msecs < 0 || !isFinite(msecs)) {
-    throw new RangeError('"msecs" argument must be ' +
-                         'a non-negative finite number');
+    throw new errors.RangeError('ERR_INVALID_OPT_VALUE', 'msecs', msecs);
```

Comment:
The motivation is to try to have minimal changes in the new error message
Suggestion (if it works):
```js
errors.TypeError('ERR_INVALID_ARG_TYPE', 'msecs',
                 'a non-negative finite number', msecs);
```